Moya.podspec

```diff
@@ -40,6 +40,6 @@ Pod::Spec.new do |s|
   s.subspec "RxSwift" do |ss|
     ss.source_files = "Sources/RxMoya/"
     ss.dependency "Moya/Core"
-    ss.dependency "RxSwift", "~> 3.3"
+    ss.dependency "RxSwift", "=> 4.0.0-beta.0"
```

Comment:
I think this should be `~> 4.0`

Reply:
I thought that `~> 4.0` doesn't contain if there's a `-alpha` or `-beta`. Let me check it again with a new project :)

Reply:
**Podfile**

```ruby
pod 'RxSwift', '4.0.0-beta.0'
pod 'Moya', :git => 'https://github.com/devxoul/Moya.git', :branch => 'rxswift4-version'
pod 'Moya/RxSwift', :git => 'https://github.com/devxoul/Moya.git', :branch => 'rxswift4-version'
```

**Result**

```
[!] Unable to satisfy the following requirements:

- `RxSwift (= 4.0.0-beta.0)` required by `Podfile`
- `RxSwift (~> 4.0)` required by `Moya/RxSwift (9.0.0)`
```

Reply:
Fair enough. I also see that you fixed the operator from `=>` to `>=` 👍

Reply:
Yeah, I'm always confused with `=>` and `>=` 😅